src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -191,22 +191,23 @@ def _add_index(in_x, parameter):
                     return {NodeType.INDEX: pos, NodeType.VALUE: item}
         else:
             return parameter
+    return None  # FIXME: what to return here?
```

Comment:
@xuehui1991 Please check this.

Reply:
Do not change the logic in this PR.

Reply:
Okay. Though it's still smelly to me.
I'll change FIXME into a note.